sdk/eventhub/Azure.Messaging.EventHubs/src/Amqp/AmqpError.cs

```diff
@@ -49,7 +49,7 @@ internal static class AmqpError
         /// <summary>Indicates that the producer requesting an operation is not allowed to publish events to the requested resource.</summary>
         public static readonly AmqpSymbol PublisherRevokedError = AmqpConstants.Vendor + ":publisher-revoked";
 
-        /// <summary>Indicates that an operation was cancelled by the Event Hubs service.</summary>
+        /// <summary>Indicates that an operation was canceled by the Event Hubs service.</summary>
```

Comment:
I prefer cancelled, but it appears one "l" is the American way. 😄

Reply:
I prefer the double as well, but my new spell checker thingy yells about it.